third_party/packages/split/lib/split.dart

```diff
@@ -88,31 +107,80 @@ Splitter flexSplit(
       gutterSize: gutterSize,
       sizes: sizes,
       minSize: minSize,
+      expandToMin: minSize?.isNotEmpty == true,
```

Comment:
[nit] https://dart.dev/guides/language/effective-dart/usage#do-use--to-convert-null-to-a-boolean-value

`expandToMin: minSize?.isNotEmpty ?? false,`

Reply:
thanks for the link. I'll adopt the standard style for this.